sdks/java/io/debezium/src/test/java/org/apache/beam/io/debezium/KafkaSourceConsumerFnTest.java

```diff
@@ -103,6 +103,7 @@ public void testStoppableKafkaSourceConsumerFn() {
         .setCoder(VarIntCoder.of());
 
     pipeline.run().waitUntilFinish();
+    // Since we're now caching calls
```

Comment:
Checking for how many times this is called will be flaky since it will all depend on how hot the cache remains, is checking how many times this is called important or should we drop this?

Reply:
not really important.